frontend/src/components/modal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,70 @@
+import { FunctionComponent, ReactNode } from 'react'
+import Button from '../Button'
+import ModalBase, { CloseReason } from './ModalBase'
+
+interface Props {
+  /**
+   * If true, the modal will be displayed.
+   */
+  isOpen?: boolean
+  /**
+   * The maximum width of the modal
+   * @default 28rem
+   */
+  modalWidth?: string | number
+  /**
+   * A title to make the context clear to the user
+   */
+  title: ReactNode
+  /**
+   * The label for the Confirm button
+   */
+  confirmLabel: ReactNode
+  /**
+   * Callback triggered when the user presses "Cancel" or the Escape key on
+   * their keyboard
+   */
+  onCancel: () => void
+  /**
+   * Callback triggered when the user confirms that they want to do the thing
+   */
+  onConfirm: () => void
+}
+
+const ConfirmationModal: FunctionComponent<Props> = ({
```

Comment:
This component forces the user to confirm an important decision. There's no "close" button and clicking the backdrop doesn't close the modal.

<img width="422" alt="Screen Shot 2021-03-25 at 10 10 37 AM" src="https://user-images.githubusercontent.com/3411183/112514332-6d913680-8d52-11eb-8d50-e1ea2c39b58c.png">